src/common/middleware/home.ts

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable prettier/prettier */
+import { BadRequestException, Injectable, NestMiddleware } from "@nestjs/common";
+import { NextFunction, Request, Response } from "express";
+
+@Injectable()
+export class homeLogger implements NestMiddleware {
```

Comment:
Again if this is for logging the request to home page, remove the token checking mechanism.